blazor/dropdown-list/grouping.md

```diff
@@ -7,7 +7,7 @@ control: DropDown List
 documentation: ug
 ---
 
-# Grouping in Blazor DropDown List Component
+# Grouping in Dropdown List
 
 The DropDownList supports wrapping nested elements into a group based on different categories. The category of each list item can be mapped through the [GroupBy](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.DropDownListFieldSettings.html#Syncfusion_Blazor_DropDowns_DropDownListFieldSettings_GroupBy) field in the data table. The group header is displayed both as inline and fixed headers. The fixed group header content is updated dynamically on scrolling the popup list with its category value.
```

Comment:
GroupBy is a complex property so we need to use like [DropDownListFieldSettings.GroupBy]